### PR TITLE
Sailfish update

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
New version was released, I will update. Please do not merge yet.

**Comment by bgruening** [comment]
Sailfish has stabilised and it's worth an update. Please review and merge if appropriate.

File: packages/package_sailfish_0_7_4/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+categories:
+- RNA
+- Tool Generators
```
Comment (nsoranzo) [inline packages/package_sailfish_0_7_4/.shed.yml]: Tool Generators?

**Comment by bgruening** [comment]
Thanks @nsoranzo fixed and updated.

**Comment by nsoranzo** [comment]
:+1:

**Pull request merged**